Source/JavaScriptCore/runtime/JSGlobalObject.h

```diff
@@ -174,7 +174,7 @@ constexpr bool typeExposedByDefault = true;
 
 #if ENABLE(WEBASSEMBLY)
 #define FOR_EACH_WEBASSEMBLY_CONSTRUCTOR_TYPE(macro) \
-    macro(WebAssemblyArray,        webAssemblyArray,        webAssemblyArray,        JSWebAssemblyArray,        Array,        object, typeExposedByDefault) \
+    macro(WebAssemblyArray,        webAssemblyArray,        webAssemblyArray,        JSWebAssemblyArray,        Array,        null,   typeExposedByDefault) \
```

Comment:
Can you make a test that you can't change the prototype either. I assume that shouldn't be possible (because of eventual shared array/structs).

Reply:
Verified and confirmed that error would be thrown when trying to set prototype of Wasm GC objects.

Reply:
Added a test.